vignettes/remote.Rmd

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
I think this vignette reads fine - I slightly wonder if a passing reference to what OrderlyWeb is, and a link to this vignette might be useful in either the Get Started, or main ReadMe. But not crucial.